src/app/mv-wrapper/media-viewer-wrapper.component.html

```diff
@@ -0,0 +1,5 @@
+<mv-media-viewer [url]="mediaURL"
```

Comment:
for my knowledge, where is this `<mv-media-viewer>` used? I can't understand how we call this thing :)